apps/example-server/src/main/application-areas/application-areas.routes.ts

```diff
@@ -0,0 +1,103 @@
+import { eq, getTableColumns } from 'drizzle-orm'
+import { Hono } from 'hono'
+import { jwt } from 'hono/jwt'
+import { z } from 'zod'
+import { db } from '../../core/db/db'
+import { applicationAreasTable } from '../../core/db/schema'
+
+const app = new Hono()
```

Comment:
Only admins can create application-areas. So use isAdmin middleware for all the below routes.